app/assets/javascripts/discourse/lib/desktop-notifications.js.es6

```diff
@@ -0,0 +1,245 @@
+
+let primaryTab;
+let liveEnabled;
```

Comment:
I would prefer it if the boolean vars are initialized to `false` rather than `null`. I know `null` is basically false but I think it helps readability when opening the file as to the type of the vars.